test/e2e/data/infrastructure-azure/v1beta1/patches/cluster-label-azuredisk-csi-driver.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+  labels:
+    azureDiskCSI: "true"
```

Comment:
nit: other labels are in the format `azure-disk-csi`

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/a67e77631aee8602206afc0be39c48cf270c3b20/templates/test/ci/patches/windows-containerd-labels.yaml#L8-L9

Reply:
Will do once the tests results are back!